index.html

```diff
@@ -0,0 +1,451 @@
+<!doctype html>
+<script src=/boot.js></script>
+<title>Composed Schema Definition Layer</title>
+
+<div id=view>
+<main markdown>
+
+<header>
+  <h1>Composed Schema Definition Layer</h1>
+  <h2>schema elements for describing composed subgraphs</h2>
+  <table class=spec-md>
+    <tr>
+      <td>status</td>
+      <td id=spec-md-status>Draft</dd>
+    </tr>
+    <tr>
+      <td>version</td>
+      <td id=spec-md-version>0.1</dd>
+    </tr>
+    <tr>
+      <td>authors</td>
+      <td id=spec-md-authors>
+        <ul class=spec-authors-list>
+          <li><a rel=author href=mailto:jake@apollographql.com>Jake Dawkins</a>
+          <li><a rel=author href=mailto:trevor@apollographql.com>Trevor Scheer</a>
+          <li><a rel=author href=mailto:ashi@apollographql.com>Ashi Krishnan</a>
+        </ul>
+      </td>
+    </tr>
+  </table>
+</header>
+
+## Introduction: What is CSDL?
+
+```mermaid
+#::[Illustration: CSDL composes multiple subgraph schemas.]
+graph LR
+  s1(astronauts.graphql)-->CSDL
+  s2(rockets.graphql)-->CSDL
+  s3(robots.graphql)-->CSDL
+  style CSDL fill:none,stroke:fuchsia,color:fuchsia;
+```
+
+The Composed Schema Definition Layer (CSDL) provides schema elements to describe a GraphQL Schema which is composed from one or more other GraphQL schemas. **Schema elements** comprise any item in a GraphQL schema—types, input types, directives, etc. CSDL defines directives and mandatory input types. 
```

Comment:
Huh, I'd been assuming that L in SDL (a term we made up, I think) and CSDL stands for Language, not Layer.

In general there is a bit of a question (although a somewhat abstract one) as to whether this is a spec for a language or for a subset of the (abstract) GraphQL type system (which can be expressed via the IDL). I think your answer is the latter; is that correct? In that case Layer does make sense, although on the other hand it does sort of confuse the issue since we do use SDL elsewhere at Apollo to mean the serialized format. Maybe instead of creating this (perhaps minor) confusion we should call this spec something else, like "Composed Schemas"? And just say that `.csdl` is a reasonable file extension to be used for when you write your Composed Schema in SDL?

Reply:
I thought this too! But then I accidentally wrote "layer" and realized that it's actually a better descriptor: the directives are more of a "layer" over the graphql idl than they are a "language"—particularly in the sense that we're not providing a grammar of any sort, just assigning meaning to particular graphql syntactic constructs.

re: naming here, what do you think of calling the spec "Composed Schema Layer," and using the `cs_` prefix? Since we have another way of detecting that a document provides a CSL (via `@using`), I don't *think* we should propose a different file extension—`.graphql` is fine (and is what tools already expect)

Reply:
I like that, or perhaps `composed_`?